### ISSUE TITLE
Tabbed Interface: Accessibility: Using Screen readers

### ISSUE THREAD

**Issue by shawnthompson** [wet-boew/wet-boew#7947]
When using NVDA and JAWS with the [tabs working examples](http://wet-boew.github.io/wet-boew/demos/tabs/tabs-en.html) in Firefox. You can't get the Screen Reader to read the contents of the tab panel. Using JAWS in IE, you can get it to read the content once you tab to the panel and hit enter to enter into the panel, the reader begins to read.

I think more tests should be done on the tabs panel plugin with screen readers.

**Comment by shawnthompson** [comment]
cc/ @andrewnordlund @rlambert27

**Comment by Bgolden67** [comment]
Anything further on this?

**Comment by waterandsewerbill2** [comment]
I looked at this in NVDA. I found I no problems in Firefox or Chrome. I have to hit enter to read the contents of the tabs, but I don't know that there is a problem with this? If this is a non-standard behavior, or not intuitive, an aria describedby attribute is a solution.

But in JAWS, one tester said the following:

"I get better results with Firefox than IE. JAWS key + ALT M works everywhere in Firefox. This is not the case with IE."
